cmd/sealer/cmd/utils/cluster.go

```diff
@@ -128,6 +126,7 @@ func constructHost(role string, joinIPs []net.IP, scaleFlags *types.Flags, clust
 	host := v2.Host{
 		IPS:   joinIPs,
 		Roles: []string{role},
+		Env:   scaleFlags.CustomEnv,
```

Comment:
sorry i dont understand  why we set CustomEnv to host level. if so, cluster level env will be lost.

Reply:
host will inherit the cluster env. And if CustomEnv is set to the global env, that means if you scale some nodes and specify env, you will overwrite the global env in ClusterFile and influence others.